.github/workflows/publish-docs.yml

```diff
@@ -37,18 +41,50 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+        id: go
+
+      - uses: actions/cache@v2
+        name: Go modules cache
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Build
+        run: make EMBEDDED_BINS_BUILDMODE=none
+
+      - name: Clean Docker
+        run: |
+          docker system prune --all --volumes --force
+
+      - name: Generate CLI docs
+        run: |
+          mkdir -p docs/cli
+          ./k0s docs markdown
```

Comment:
might be faster to simply run the docs command without compiling the whole binary with `go run main.go docs markdown`

Reply:
Nice, didn't know `go run` works for some commands.